src/databricks/labs/remorph/reconcile/recon_capture.py

```diff
@@ -80,24 +77,37 @@ def _write_df_to_delta(df: DataFrame, table_name: str, mode="append"):
         raise WriteToTableException(message) from e
 
 
-def generate_final_reconcile_output(recon_id: str, spark: SparkSession) -> ReconcileOutput:
+def _get_db_prefix(metrics: ReconcileMetricsConfig) -> str:
+    return f"{metrics.catalog}.{metrics.schema}"
```

Comment:
If add a condition catalog name local unit test with some unique identifier we allow to return only schema name this will ensure we don't need to monkey patch 'tech debt'